tests/virt/cluster/general/test_mass_machine_type_transition.py

```diff
@@ -0,0 +1,218 @@
+            {"name": "NAMESPACE", "value": namespace.name},
+        ],
+        "securityContext": {
+            "allowPrivilegeEscalation": False,
+            "capabilities": {"drop": ["ALL"]},
+            "privileged": False,
+            "runAsNonRoot": True,
+            "seccompProfile": {"type": "RuntimeDefault"},
+        },
+    }
+
+    with Job(
+        name=KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+        namespace=kubevirt_api_lifecycle_namespace.name,
+        client=admin_client,
+        containers=[container],
+        service_account=KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+        restart_policy="Never",
+        backoff_limit=0,
+    ) as job:
+        job.wait_for_condition(
+            condition=Job.Condition.COMPLETE,
+            status=Job.Condition.Status.TRUE,
+            timeout=TIMEOUT_2MIN,
+        )
+        yield job
+
+
+@pytest.fixture(scope="session")
+def kubevirt_api_lifecycle_namespace(admin_client):
+    yield from create_ns(name=KUBEVIRT_API_LIFECYCLE_AUTOMATION, admin_client=admin_client)
+
+
+@pytest.fixture(scope="session")
+def kubevirt_api_lifecycle_service_account(kubevirt_api_lifecycle_namespace):
+    with ServiceAccount(name=KUBEVIRT_API_LIFECYCLE_AUTOMATION, namespace=kubevirt_api_lifecycle_namespace.name) as sa:
+        add_scc_to_service_account(
+            namespace=KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+            scc_name="privileged",
+            sa_name=sa.name,
+        )
+        yield sa
+
+
+@pytest.fixture(scope="session")
+def kubevirt_api_lifecycle_cluster_role_binding(admin_client, kubevirt_api_lifecycle_service_account):
+    with ClusterRoleBinding(
+        name=KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+        cluster_role="cluster-admin",
+        client=admin_client,
+        subjects=[
+            {
+                "kind": ServiceAccount.kind,
+                "name": KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+                "namespace": KUBEVIRT_API_LIFECYCLE_AUTOMATION,
+            }
+        ],
+    ) as crb:
+        yield crb
+
+
+@pytest.fixture()
+def vm_with_schedulable_machine_type(admin_client, namespace):
+    name = "vm-with-schedulable-machine-type"
+    with VirtualMachineForTests(
+        name=name,
+        namespace=namespace.name,
+        client=admin_client,
+        image=Images.Rhel.RHEL9_REGISTRY_GUEST_IMG,
+        memory_guest=Images.Rhel.DEFAULT_MEMORY_SIZE,
+        machine_type=MachineTypesNames.pc_q35_rhel8_1,
+    ) as vm:
+        running_vm(vm=vm)
+        yield vm
+
+
+@pytest.fixture()
+def vm_with_unschedulable_machine_type(admin_client, namespace):
+    with VirtualMachineForTests(
+        name="vm-with-unschedulable-machine-type",
+        namespace=namespace.name,
+        client=admin_client,
+        image=Images.Rhel.RHEL9_REGISTRY_GUEST_IMG,
+        memory_guest=Images.Rhel.DEFAULT_MEMORY_SIZE,
+        machine_type=MachineTypesNames.pc_q35_rhel7_4,
+    ) as vm:
+        vm.start(wait=False)
```

Comment:
```suggestion
        vm.start()
```
wait is false by default here